web/controllers/user_controller.ex

```diff
@@ -38,28 +38,49 @@ defmodule Pairmotron.UserController do
   end
 
   def update(conn, %{"id" => id, "user" => user_params}) do
+    update_user(conn, id, user_params, :show)
+  end
+
+  def delete(conn, %{"id" => id}) do
+    user = Repo.get!(User, id)
+
+    # Here we use delete! (with a bang) because we expect
+    # it to always work (and if it does not, it will raise).
+    Repo.delete!(user)
+
+    conn
+    |> put_flash(:info, "User deleted successfully.")
+    |> redirect(to: user_path(conn, :index))
+  end
+
+  def activate(conn, %{"user_id" => id}) do
+    update_user(conn, id, %{active: true}, :index)
+  end
+
+  def deactivate(conn, %{"user_id" => id}) do
+    update_user(conn, id, %{active: false}, :index)
+  end
```

Comment:
We should definitely explore ways to accomplish this without adding new non-RESTful controller methods, as discussed.